### PR TITLE
@l2succes => [WIP] Working price slider

### PR DESCRIPTION
**Pull request by mzikherman**
This isn't styled at all, (and I'm not sure how to yet, I'm just importing the base CSS), but it works!

### CODE REVIEW

File: src/Apps/Collect/Components/ArtworkGrid/index.tsx [hunk 1]
```diff
@@ -2,6 +2,8 @@ import { Sans } from "@artsy/palette"
 import { ArtworkGrid_viewer } from "__generated__/ArtworkGrid_viewer.graphql"
 import { FilterState } from "Apps/Collect/FilterState"
 import { ContextConsumer } from "Artsy"
+import { Range } from "rc-slider"
+import "rc-slider/assets/index.css"
```
Comment (mzikherman) [inline src/Apps/Collect/Components/ArtworkGrid/index.tsx]: Without this, the CSS isn't brought in and the slider doesn't even appear visible. With this line, it looks like something:

screen shot 2018-09-05 at 8 28 38 pm

File: src/Apps/Collect/Components/ArtworkGrid/index.tsx [hunk 2]
```diff
@@ -134,6 +136,29 @@ class Filter extends Component<Props> {
                                   {!hideTopBorder && <Separator mb={1} />}
 
                                   {this.renderWaysToBuy(filters, mediator)}
+
+                                  <div>
+                                    <Range
```
Comment (mzikherman) [inline src/Apps/Collect/Components/ArtworkGrid/index.tsx]: Still need to update text above the slider to show you the range you've selected

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit b4341da94011438f3365cca7e5e304e08fd209eb

https://deploy-preview-1215--artsy-reaction.netlify.com

File: src/Apps/Collect/Components/ArtworkGrid/index.tsx (hunk 1 above)
Comment (l2succes) [reply src/Apps/Collect/Components/ArtworkGrid/index.tsx]: we could wrap the whole stylesheet (https://github.com/react-component/slider/blob/master/assets/index.less) into a styled component and just convert the LESS in css-in-js

**Pull request merged**